react/package.json

```diff
@@ -43,11 +43,12 @@
     "babel-jest": "^24.1.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react-app": "^7.0.1",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme": "^3.4.1",
+    "enzyme-adapter-react-16": "^1.2.0",
     "enzyme-to-json": "^3.3.5",
     "jest": "^24.1.0",
-    "jest-transform-graphql": "^2.1.0"
+    "jest-transform-graphql": "^2.1.0",
+    "identity-obj-proxy": "^3.0.0"
```

Comment:
Needed?

Reply:
Yeah, this mocks the css modules for my tests